pkg/util/push/otel.go

```diff
@@ -67,7 +65,7 @@ func OTLPHandler(
 		}
 
 		if r.ContentLength > int64(maxRecvMsgSize) {
-			return nil, httpgrpc.Errorf(http.StatusRequestEntityTooLarge, messageSizeLargerErrFmt, r.ContentLength, maxRecvMsgSize)
+			return nil, httpgrpc.Errorf(http.StatusRequestEntityTooLarge, "received message larger than max (%d > %d)", r.ContentLength, maxRecvMsgSize)
```

Comment:
Inlined it because this string is not used anywhere else. Should this be a `MsgSizeTooLargeErr` as well? Not sure how to do that as it's an error from `httpgrpc`

Reply:
I think this should be a `newDistributorMaxWriteMessageSizeErr()`, no? Can we also improve the assertions in `TestHandler_otlpWriteRequestTooBigNoCompression()` and `TestHandler_otlpWriteRequestTooBigWithCompression()`?